source/FreeRTOS_DHCPv6.c

```diff
@@ -459,6 +395,18 @@ void vDHCPv6Process( BaseType_t xReset,
             else
             {
                 FreeRTOS_printf( ( "vDHCPv6Process: malloc failed %u bytes\n", ( unsigned int ) sizeof( *pxEndPoint->pxDHCPMessage ) ) );
+

```

Comment:
Why is it added now?

Reply:
As [comment](https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/pull/857/files#r1185777339) below, to stop all DHCPv6 flow when it fail to allocate buffer for DHCP message.